tools/doxygen/doxygen.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+LAMMPSBASE=${PWD} ;
+LAMMPSSRC="${LAMMPSBASE}/src" ;
+LAMMPSDOC="${LAMMPSBASE}/doc" ;
+LAMMPSDOCHTML="${LAMMPSDOC}/html" ;
+LAMMPSDEVELOPERDOC="${LAMMPSBASE}/doc/src/Developer" ;
+LAMMPSDOCTMP="/tmp/lammps-docs-*" ;
+LAMMPSDOCMAKE="${LAMMPSDOC}/Makefile" ;
+LAMMPSDOXYGEN="${LAMMPSBASE}/tools/doxygen" ;
+LAMMPSDOXYGENDOC="${LAMMPSDOXYGEN}/doc" ;
+LAMMPSDOXYGENDOCMANUAL="${LAMMPSDOXYGEN}/doc/html/Manual" ;
+
+LAMMPSDOXYFILE=${LAMMPSDOXYGEN}/Doxyfile.lammps ;
+DOXYFILE=${LAMMPSDOXYGEN}/Doxyfile ;
+
+LAMMPSDEVELOPERDOXFILE=${LAMMPSDOXYGEN}/Developer.dox.lammps ;
+DEVELOPERDOXFILE=${LAMMPSDOXYGEN}/Developer.dox ;
+
+FIG2DEV=`which fig2dev` ;
+DOXYGEN=`which doxygen` ;
+
+
+error=0;
+
+
+if [[ -d ${LAMMPSSRC} && -d ${LAMMPSDEVELOPERDOC} && -d ${LAMMPSDOC} && -f ${LAMMPSDOCMAKE} && -d ${LAMMPSDOXYGEN} && -f ${LAMMPSDOXYFILE} ]] ;
+ then
+   cd ${LAMMPSSRC} ;
+   make no-all ;
+   cd ${LAMMPSBASE} ;
+ else
+  error=1 ;
+  echo "Cannot configure LAMMPS sources - Please run doxygen.sh from the LAMMPS root directory."
+ fi ;
+
+
+if [ ${error} == 0 ] ;
+  then
+    if [[ ! -x ${FIG2DEV} ]] ;
+      then
+        error=1 ;
```

Comment:
I would do an `exit 1` here and remove all the `[ ${error} = 0 ]` below.

Reply:
I simplify the shell script with exits.